usaspending_api/settings.py

```diff
@@ -34,7 +34,7 @@
 SECRET_KEY = get_random_string()
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
We need to revert this.

Reply:
Can we just add a linter rule that treats `DEBUG = True` as badly written code?

Reply:
Another viable option is leaving it set to True in the repo and having the deploy pipeline set it to False.